packages/cli/src/services/config.ts

```diff
@@ -21,12 +21,14 @@ enum Env {
 
 const config = {
   auth: new Conf({
+    projectName: '@checkly/cli',
```

Comment:
@clample , isn't it possible to create a fallback logic? I mean, first check `projectName: 'checkly'`, then the legacy one. If not, users with the new version will still being storing their config in the old project name.
Another option could be to store config in both `projectNames` making easier to deprecate the `@checkly/cli` project name in the future.

Reply:
@nahuelon that might be possible. For now, though, I don't see any downsides to just only storing the config in the `@checkly/cli` file.